src/view/reviewManager.ts

```diff
@@ -482,6 +482,7 @@ export class ReviewManager {
 	private async getPullRequestData(pr: PullRequestModel & IResolvedPullRequestModel): Promise<void> {
 		try {
 			this._comments = await pr.getReviewComments();
+			await pr.getReviewThreads();
```

Comment:
Value not saved anywhere?

Reply:
Changed this to use a new `initializeReviewThreadsCache` function to hopefully make its purpose clearer